src/components/Search/ResultCard.tsx

```diff
@@ -1,17 +1,36 @@
+import { Checkbox, Grid } from "@trussworks/react-uswds";
+import { useContext } from "react";
 import { useTranslation } from "react-i18next";
 import { Link, useLocation } from "react-router-dom";
+import { CompareContext } from "../../pages/Search/Search";
 
 import { CareProviderSearchResult } from "../../types";
 import BasicResultDetail from "../ResultDetail/BasicResultDetail";
 import MilesAway from "./MilesAway";
 
 type ResultCardProps = {
   data: CareProviderSearchResult;
+  isMobile?: boolean;
 };
 
-export default function ResultCard({ data }: ResultCardProps) {
+export default function ResultCard({ data, isMobile }: ResultCardProps) {
+  const { providers, setProviders } = useContext(CompareContext);
```

Comment:
nit: i got a lil confused scanning code, may be clearer to name this something along the lines selectedCompareProviders